apps/src/code-studio/pd/workshop_dashboard/reports/organizer_report.jsx

```diff
@@ -0,0 +1,241 @@
+/**
+ * Organizer Report
+ */
+import React from "react";
+import ReportTable from "./report_table";import FontAwesome from '@cdo/apps/templates/FontAwesome';
+import {
+  Checkbox,
+  Button
+} from 'react-bootstrap';
+
+const FACILITATOR_DETAILS_COUNT = 6;
+const ATTENDANCE_DAYS_COUNT = 5;
+const QUERY_URL = "/api/v1/pd/workshop_organizer_report";
+
+const styles = {
+  link: {cursor: 'pointer'}
+};
+
+const OrganizerReport = React.createClass({
+  propTypes: {
+    startDate: React.PropTypes.string.isRequired,
+    endDate: React.PropTypes.string.isRequired,
+    queryBy: React.PropTypes.string.isRequired
```

Comment:
if there are only two allowed values, you could use `React.PropTypes.oneOf`([example](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/applab/AppLabView.jsx#L22))